APIs/schemas/receiver_core.json

```diff
@@ -42,19 +42,19 @@
           }
         },
         "subscription": {
-          "description": "Object containing the 'sender_id' currently subscribed to.",
+          "description": "Object indicating how this Receiver is currently configured to receive data.",
           "type": "object",
           "required": ["sender_id", "active"],
           "properties": {
             "sender_id": {
               "type": ["string", "null"],
-              "description": "UUID of the Sender that this Receiver is currently subscribed to",
+              "description": "UUID of the Sender from which this Receiver is currently configured to receive data, if it is active and receiving from an NMOS Sender, otherwise null",
```

Comment:
```suggestion
              "description": "UUID of the Sender from which this Receiver is currently configured to receive data. Only set if it is active and receiving from an NMOS Sender; otherwise null.",
```